docker-build.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash -e
```

Comment:
Why are we creating a shell script for this?  If anything we should add a subcommand to etest for this and skip the shell script.  That'll keep it consistent and easier to test as well.

Reply:
I think this would be difficult to do in python and that the end user shouldnt need to use this, they should be able to pull from dockerhub. However, if you think a python implementation would be painless Im open to it.